### DISCUSSION TITLE
zh/algorithms/hashtable

### DISCUSSION THREAD

**Discussion by giscus** [selfboot/ai_gallery#12] [Q&A]
# zh/algorithms/hashtable

哈希表用于高效地存储键值对，本页面采用网格布局模拟哈希表的内部结构，可视化展示了哈希表的基本操作，包括插入、删除、搜索等。支持选择不同 hash 函数以及设置 hash 表大小，帮助理解哈希表的工作原理。

https://gallery.selfboot.cn/zh/algorithms/hashtable

**Comment by HanHuiOnLine** [comment]
这是个加深理解的好方法
